src/extension-installations/Actions.js

```diff
@@ -0,0 +1,68 @@
+import rio, { update as rioUpdate, find, remove as rioRemove } from '@shoutem/redux-io';
+import { mergeSettings, getSettings } from '../services/settings';
+import extensionInstallationResource, { INSTALLATIONS } from '../resources/extensionInstallations';
+
+export default class Actions {
```

Comment:
Can we refactor this to remove this class? It would be nice if the entire redux layer only exports functions: action creators, selectors, ...

We could capture the config in a closure when the api is being initialized, and export all actions from there. That would allow us to move the logic that resolves the context to a single place, instead of doing this in each action creator. The `Api` could then first resolve the context, and then call the appropriate action with the resolved context.